qa/suites/orch/cephadm/smb/tasks/deploy_smb_mgr_ctdb_res_basic.yaml

```diff
@@ -0,0 +1,135 @@
+roles:
+# Test is for basic smb deployment & functionality. one node cluster is OK
+- - host.a
+  - mon.a
+  - mgr.x
+  - osd.0
+  - osd.1
+  - client.0
+- - host.b
+  - mon.b
+  - osd.2
+  - osd.3
+- - host.c
+  - mon.c
+  - osd.4
+  - osd.5
+# Reserve a host for acting as an smb client
+- - host.d
+  - cephadm.exclude
+overrides:
+  ceph:
+    log-only-match:
+      - CEPHADM_
+tasks:
+- cephadm.configure_samba_client_container:
+    role: host.d
+- cephadm:
+
+- cephadm.shell:
+    host.a:
+      - ceph fs volume create cephfs
+- cephadm.wait_for_service:
+    service: mds.cephfs
+
+- cephadm.shell:
+    host.a:
+      # add subvolgroup & subvolumes for test
+      - cmd: ceph fs subvolumegroup create cephfs smb
+      - cmd: ceph fs subvolume create cephfs sv1 --group-name=smb --mode=0777
+      - cmd: ceph fs subvolume create cephfs sv2 --group-name=smb --mode=0777
+      # set up smb cluster and shares
+      - cmd: ceph mgr module enable smb
+      # TODO: replace sleep with poll of mgr state?
```

Comment:
if you do want to do this, `ceph mgr module ls` can return json including the list of enabled modules
```
[ceph: root@vm-00 /]# ceph mgr module ls -f json | jq .enabled_modules
[
  "cephadm",
  "dashboard",
  "iostat",
  "nfs",
  "prometheus",
  "restful"
]
[ceph: root@vm-00 /]# 
[ceph: root@vm-00 /]# ceph mgr module enable smb
[ceph: root@vm-00 /]# 
[ceph: root@vm-00 /]# ceph mgr module ls -f json | jq .enabled_modules
[
  "cephadm",
  "dashboard",
  "iostat",
  "nfs",
  "prometheus",
  "restful",
  "smb"
]
```
so something like
```
[ceph: root@vm-00 /]# ceph mgr module disable smb
[ceph: root@vm-00 /]# 
[ceph: root@vm-00 /]# ceph mgr module ls -f json | jq .enabled_modules | jq -r .[] | grep smb
[ceph: root@vm-00 /]# echo $?
1
[ceph: root@vm-00 /]# ceph mgr module enable smb
[ceph: root@vm-00 /]# 
[ceph: root@vm-00 /]# ceph mgr module ls -f json | jq .enabled_modules | jq -r .[] | grep smb
smb
[ceph: root@vm-00 /]# echo $?
0
```
could work

Reply:
Thanks. This is a copy and paste line from the other smb yamls that all do the same thing. In the future I can take a pass over the test yaml and make it better but I don't think fixing this todo is needed for this pr.